docs/oauth.md

```diff
@@ -13,46 +35,64 @@ To enable OAuth for your connector add an `<oauth-config>` field in the manifest
   manifest.xml
 
   <?xml version='1.0' encoding='utf-8' ?>
-
   <connector-plugin class=...>
     ...
     <dialect file='dialect.tdd'/>
-    <oauth-config file='oauthConfig.xml'/>    <!-- add this to to define your OAuth Configs -->
+    <oauth-config file='oauthConfigPing.xml'/>    <!-- add this to to define your OAuth Configs -->
   </connector-plugin>
 ```
 
-In your connectionFields.xml file make sure add an `authentication` field with a value equal to `oauth`.
+Starting in Tableau 2023.1, you can add multiple OAuth configs, embedded in the plugin. The end users may also provide external/custom OAuth configurations:
+- By installing the config files in the Tableau directory. See [Create Site OAuth Client](#create-site-oauth-client-20231)
+- By uploading the config for a site level OAuth client. See [External OAuth Config on Desktop](#externalcustom-oauth-configs-on-desktop)
```

Comment:
The link here needs to be switched with the link above, i.e 
`By uploading the config for a site level OAuth client. See [Create Site OAuth Client](#create-site-oauth-client-20231)`

Reply:
Good catch